pls_core/src/inflections/mod.rs

```diff
@@ -134,9 +134,17 @@ fn get_pali1_metadata(pali1: &str, host: &dyn PlsInflectionsHost) -> Result<Pali
 
         pm.inflection_class = inflection_class_from_str(inflection_class);
         pm.like = if !like.is_empty() {
-            format!("like {}", host.transliterate(like)?)
+            format!(
+                "{} like {}",
+                if inflection_class == "verb" {
+                    "conjugation"
+                } else {
+                    "declension"
+                },
```

Comment:
which part of the fix is this?

Reply:
my bad. I should've looked at this revision - 
![image](https://user-images.githubusercontent.com/60284375/115949375-66e11500-a4f2-11eb-9ce4-38e9a096cb6a.png)